ci/nixpkgs-pr/build-context/commit-message.md

```diff
@@ -13,4 +13,4 @@ Update to latest upstream version
  * [x]  Tested execution of all binary files (usually in `./result/bin/`)
  * [ ]  Determined the impact on package closure size (by running `nix path-info -S` before and after)
  * [ ]  Ensured that relevant documentation is up to date
- * [x]  Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).
```

Comment:
This change should not be needed anymore once you change the base to `develop` + rebase onto `develop` (since I've already changed the link on `develop`)